docs/reference/early-access.md

```diff
@@ -0,0 +1,54 @@
+---
+id: early-access
+title: Early access
+sidebar_label: Early access
+description: "Use early access releases to quickly learn about upcoming features and components."
+---
+
+Use early access releases to quickly learn about upcoming features and components.
+
+## Types of early access releases
+
+Selected Camunda features and components are released upfront as alpha or beta versions, to provide users the opportunity to test a feature and participate in the development by giving feedback prior to market launch (general availability, GA).
+
+Early access releases are made available outside of the [release policy](/reference/release-policy.md).
+
+### Alpha
+
+Alpha features and components are introduced to gather early feedback from users. These features and components are still under heavy development, and are expected to change. These releases are intended only for non-production usages in trying out recent, potentially yet unfinished new features. Characteristics are:
+
+- Not for production use
+- APIs, dependencies and configuration are likely to change
+- Not necessarily feature-complete
+- May not be documented
+- Updatability to a newer release not guaranteed
+- Support based on SLAs agreed with you, but bugs are treated with the same priority as feature or help requests
+- No maintenance service
+- On SaaS: no availability targets
+
+### Beta
+
+The scope of beta features and components is close enough to the targeted GA release that we are willing to provide update support, but we're still making the final changes. Therefore, it may still be missing some features. It comes with similar characteristics as an alpha release, but with following differences:
```

Comment:
Is it not like this

```suggestion
The scope of beta features and components is close enough to the targeted GA release that we are willing to provide update support, but we're still making the final changes. Therefore, it may still be missing some features. It comes with the same characteristics as an alpha release, except for the following differences:
```

WDYT?

Reply:
Agree, this is better and clearer wording